### PR TITLE
remove logs.post() and logs.postPaths() from Zealot

### PR DESCRIPTION
**Pull request by nwt**
The corresponding Zed API routes are going away, so replace calls
with logs.add()/logs.commit().

### CODE REVIEW

File: plugins/log-loader/index.ts [hunk 1]
```diff
@@ -53,4 +49,17 @@ export const activate = (api: BrimApi) => {
   })
 }
 
+function nodeJSReadableStreamFromReadableStream(
```
Comment (nwt) [inline plugins/log-loader/index.ts:52]: Style-wise, should this anonymous (`const nodeJSReadableStreamFromReadableStream = ...`) rather than named?

Comment (nwt) [reply plugins/log-loader/index.ts:52]: Also, I'd like to acknowledge that this is ugly. If there's a way to avoid it that I missed, let me know!

Comment (jameskerr) [reply plugins/log-loader/index.ts:52]: Nope, named is still cool.

**Review by jameskerr — COMMENTED** [review]
Thanks for converting this!

**Review by jameskerr — APPROVED** [review]

**Pull request merged**